rosidl_generator_type_description/package.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rosidl_generator_type_description</name>
+  <version>3.3.1</version>
+  <description>Generate hashes and descriptions of ROS 2 interface types, per REP-2011.</description>
+
+  <maintainer email="me@emersonknapp.com">Emerson Knapp</maintainer>
+
+  <license>Apache License 2.0</license>
+
+  <author email="me@emersonknapp.com">Emerson Knapp</author>
+
+  <buildtool_depend>ament_cmake_python</buildtool_depend>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
+
+  <buildtool_export_depend>ament_cmake_core</buildtool_export_depend>
+  <buildtool_export_depend>python3</buildtool_export_depend>
```

Comment:
You know, I almost wonder if this package should have a `buildtool_export_depend` on `type_description_interface`.  It is true it doesn't *directly* depend on it (because we are basically making a copy of some of the structure of it), but it sort of indirectly does.  Thoughts?

Reply:
Update: that _is_ a circular dependency, the exact one I'm jumping through hoops to get around with the struct-copying in the next fun PR after this one.

Reply:
OK, got it.  I thought that might be the case, but I was just double-checking.  Thanks, no action needed here.